### ISSUE TITLE
📚 Documentation: skip to content links need adding [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#278]
## Issue
The docs are multi page, so we should have skip to main content link on each page.

![skip to content link on W3C site](https://github.com/appwrite/website/assets/121672086/8efb1d15-c14f-4dfc-b4eb-483115a8bb17)

## Why it matters
On multi-page sites (vs Single Page Application (SPA)) it is advisable to have "skip to content" links as they reduce the number of tab stops required to get to the page content each time you navigate to a new page.

Otherwise you have to tab past the whole menu to get to the page content. 

## Suggested fix
Add skip links, similar to the [example skip links codepen](https://codepen.io/GrahamTheDev/pen/rNPVobP) here.

These are links that are not visible unless they receive keyboard focus.

## Relevant Success Criterion WCAG
[Bypass Blocks in WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

## Additional Notes
Should be added to the main template / layouts as the first link on the page. Should come before any other active content on the page. 

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Nwakaego-Ego** [comment]
Hi Graham,

I came across your tweet regarding the issues you raised and I'm eager to contribute. I noticed that the Appwrite Website is built with Svelte and SvelteKit. Although I'm a junior React developer, I'm willing to learn and work on this project. Could you please let me know if this opportunity is open to beginners like me?

If not will it be possible to shadow anyone working on this project?

Your feed back is highly appreciated.

Thank you

**Comment by ketanbaitule** [comment]
Hi, I would like to work on this

**Comment by ketanbaitule** [comment]
Hi, The [Svelte doc](https://svelte.dev/docs/introduction) Skip To Content Link First Goes To Sidebar, And then to content.

We have 2 Options: Just follow the svelte one, or medium one which directly goes to main content first.

What do you want me to do so??

**Comment by GrahamTheDevRel** [comment]
Hey @ketanbaitule Svelte Docs do it incorrectly as the point is to bypass the very long navigation. 

So go to the main content. 👍🏼💗

**Comment by EVDOG4LIFE** [comment]
@GrahamTheDevRel - wanted to circle back on this. It appears that https://github.com/appwrite/website/pull/359/files was merged in December and the skip to content is live. Are you able to verify that this feature was implemented appropriately?

**Comment by EVDOG4LIFE** [comment]
Closing out as related PRs are merged, and skip to content links are active. Please reopen or create a new issue if more is needed.
